packages/fxa-auth-server/fxa-oauth-server/lib/db/mysql/patches/patch-026-027.sql

```diff
@@ -0,0 +1,7 @@
+-- The `profileChangedAt` column was removed in patch-023-024 because
+-- it caused migration issues. Now that we have resolved our migration
+-- issues, it is safe to add this column back.
+ALTER TABLE tokens ADD COLUMN profileChangedAt BIGINT DEFAULT NULL,
```

Comment:
Can you add this to [schema.sql](https://github.com/mozilla/fxa/blob/68978f34c9ed96536a6adc490f6d2f6a178d34b6/packages/fxa-auth-server/fxa-oauth-server/lib/db/mysql/schema.sql#L47) just so it's documented? Or is that file generated?

Reply:
Looks like this was added below, or do you mean another file?

Reply:
No, I just wasn't being thorough enough.